ang/taskRunConfiguration/taskRunConfiguration.html

```diff
@@ -0,0 +1,41 @@
+<div class="sql-task-task-run-configuration">
+    <div class="crm-form-block">
+        <div class="sql-task-task-run-configuration-block">
+            <h1 crm-page-title>{{ts('Run this task?')}}</h1>
+            <div class="help sql-task-no-margin">
+                <span>{{ts("Are you sure you want to run this task?")}}</span>
```

Comment:
The run prompt is a bit weird now. First you go to the page where it says "Are you sure you want to run this task?". Then, after clicking, you get redirected to the run page which says "Please use the "run again" button below, if you want to execute this task again.". Only after clicking again will the task execute.

Maybe we can just re-use the old page? If we add a different parameter, we can switch between the `Please use the "run again" button below, if you want to execute this task again.` and `Are you sure you want to run this task?` text depending on the value of that parameter. Other than the message, the page looks like it does pretty much the same thing - you either want to run, or go back to configure/manage.